app/scripts/views/mixins/password-strength-mixin.js

```diff
@@ -82,6 +84,7 @@ define(function (require, exports, module) {
             if (passwordCheckStatus === 'BLOOMFILTER_HIT' ||
```

Comment:
@vbudhram Logging by whitelisting, only passwords that are too long, or not found to be common are *not* logged, everything else is.